proxy/proxy.go

```diff
@@ -213,6 +221,12 @@ func mapRequest(r *http.Request, rt *routing.Route, host string) (*http.Request,
 	u.Scheme = rt.Scheme
 	u.Host = rt.Host
 
+	backendURL, err := url.Parse(rt.Backend)
```

Comment:
I checked it again and you are right. We have not to copy the User from URL. I can drop the backendURL from mapRequest.